src/client/store/ui/geo/hooks/useAgreementLevel.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react'
+
+import { LayerKey, LayerSectionKey } from '@meta/geo'
+
+import { useAppDispatch } from '@client/store'
+import { useCountryIso, usePrevious } from '@client/hooks'
+
+import { GeoActions } from '../slice'
+import { useGeoLayerSection } from '.'
+
+export const useAgreementLevel = (sectionKey: LayerSectionKey, layerKey: LayerKey) => {
```

Comment:
This hooks doesn't belong to store. It dispatches an action . If it's used in one please , better to put in a hook folder where used. If common hook , under common hook folder

Reply:
Done, thanks!